### DISCUSSION TITLE
Soft wrapping on mobile web site diff view

### DISCUSSION THREAD

**Discussion by dead10ck** [community/community#42564] [Pull Requests] [answered]
It seems on the mobile web site's diff view, soft wrapping has been turned on, and as far as I can tell, there's no way to turn it off. This makes diffs very hard to read on small screens as the shape of the code is completely destroyed.

![Screenshot_20221224-104050_Firefox](https://user-images.githubusercontent.com/846275/209442884-02c7dcbe-bc69-4b69-80a3-3eb9ebfbf2be.png)

**Comment by AliAryanTech** [comment, marked as answer]
Same issues we're talking about - [Here](https://github.com/orgs/community/discussions/42298)

**Reply by dead10ck** [reply]
Ah, thanks! I wasn't able to find this with a search.
